### PR TITLE
Add rudimentary version of helper classes

### PR DESCRIPTION
**Pull request by fwege**
I think it makes sense to pull a rudimentary version of the helper classes into the main branch, soon.

ToDos before merging:
- Check interaction with variables defined in `.env`
- Update dependencies in `pyproject.toml`

### CODE REVIEW

File: python/seguro/common/MQTTclient.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import paho.mqtt.client as mqtt
+from queue import Queue
+
+class MQTTclient:
```
[the remark this answers is not in the document]
Comment (fwege) [reply python/seguro/common/MQTTclient.py]: Fixed in 12f7325d12864c0eb1e90451517bb8d373ae40bf

File: python/seguro/common/S3Storage.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
+from minio import Minio
+import io
+class S3Storage:
```
[the remark this answers is not in the document]
Comment (fwege) [reply python/seguro/common/S3Storage.py:5]: Fixed in 12f7325d12864c0eb1e90451517bb8d373ae40bf

File: python/seguro/common/S3Storage.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    def __init__(self, server, port, access_key, secret_key):
+        self.tracked_files = {}
+        self.client = self.__connect(server, port, access_key, secret_key)
+
+
```
[the remark this answers is not in the document]
Comment (fwege) [reply python/seguro/common/S3Storage.py]: Fixed in 12f7325d12864c0eb1e90451517bb8d373ae40bf

File: python/seguro/common/S3Storage.py [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    def get_file(self, filename, file, bucket="seguro"):
+        if not self.client.bucket_exists(bucket):
+            print(f"Error: Bucket \"{bucket}\" does not exist...")
+            return
+
+        self.client.fget_object(
+            bucket, file, filename
+        )
+
```
[the remark this answers is not in the document]
Comment (fwege) [reply python/seguro/common/S3Storage.py:26]: What do you mean?

File: python/seguro/common/S3Storage.py [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    def write_to_file(self, filename, message, bucket="seguro"):
+        if not self.client.bucket_exists(bucket):
+            print(f"Error: Bucket \"{bucket}\" does not exist...")
+            return
+
+        self.client.put_object(
+            bucket, filename, io.BytesIO(b'%b' % message.encode('utf8')), len(message)
+        )
+
+    def file_changed(self, filename, bucket="seguro"):
```
[the remark this answers is not in the document]
Comment (stv0g) [reply python/seguro/common/S3Storage.py:45]: See also #10

File: python/seguro/common/S3Storage.py [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+
+    def __connect(self, SERVER, PORT, ACCESS_KEY, SECRET_KEY):
+        client = Minio(endpoint=SERVER+":"+str(PORT),
+                    access_key=ACCESS_KEY,
+                    secret_key=SECRET_KEY,
+                    secure=False)
+        return client
+
+    def get_file(self, filename, file, bucket="seguro"):
```
[the remark this answers is not in the document]
Comment (stv0g) [reply python/seguro/common/S3Storage.py:18]: Lets address this in a separate PR. See #18

File: python/seguro/common/MQTTclient.py [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+        self.client.subscribe(topic)
+
+    def start_listening(self):
+        self.client.loop_start()
+
+    def stop_listening(self):
+        self.client.loop_stop()
+
+    def publish(self, topic, message):
+        self.client.publish(topic, message)
```
[the remark this answers is not in the document]
Comment (stv0g) [reply python/seguro/common/MQTTclient.py]: I think the file has no newline at the end?

File: python/seguro/common/S3Storage.py [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+        if not self.client.bucket_exists(bucket):
+            print(f"Error: Bucket \"{bucket}\" does not exist...")
```
[the remark this answers is not in the document]
Comment (fwege) [reply python/seguro/common/S3Storage.py]: I opened an issue for that #27

**Review by stv0g — APPROVED** [review]

**Pull request merged**